### PR TITLE
Restrict credentials in config to Basic scheme

### PR DESCRIPTION
**Pull request by jmyersmsft**

### CODE REVIEW

**Comment by leantk** [comment]
Do we think this change breaking any existing customers? I don't fully grasp the change if you could give a little more detail

**Comment by bishal-pdMSFT** [comment]
On the same line as Leah's question - does it break existing auth behavior for .Net Core CLI task?

**Comment by jmyersmsft** [comment]
@leantk @bishal-pdMSFT https://github.com/NuGet/NuGet.Client/pull/2297

File: Tasks/Common/packaging-common/nuget/NuGetConfigHelper2.ts [hunk 1]
```diff
@@ -84,6 +84,7 @@ export class NuGetConfigHelper2 {
         }
 
         sources.forEach((source) => {
+            tl.debug(`considering source ${source.feedUri}. Internal: ${source.isInternal}`)
```
Comment (johnterickson) [inline Tasks/Common/packaging-common/nuget/NuGetConfigHelper2.ts:87]: I love this logging!

**Pull request merged**